Shift-Demo/Shift-Demo/SplitTransitionAnimatedViewController.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  SplitTransitionAnimatedViewController.swift
+//  Shift-Demo
+//
+//  Created by Matthew Buckley on 12/19/15.
+//  Copyright © 2015 Raizlabs. All rights reserved.
+//
+
+import UIKit
+import Shift
+
+class SplitTransitionAnimatedViewController: UITableViewController {
+
+    var initialNavigationControllerDelegate: UINavigationControllerDelegate?
+    var currentTransition: SplitTransition?
+    var currentCell: UITableViewCell?
+
+    let colors = [
+        UIColor.redColor(),
+        UIColor.purpleColor(),
+        UIColor.blueColor(),
+        UIColor.greenColor(),
+        UIColor.yellowColor(),
+        UIColor.orangeColor(),
+        UIColor.redColor(),
+        UIColor.purpleColor(),
+        UIColor.blueColor(),
+        UIColor.greenColor(),
+    ]
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        tableView.registerClass(UITableViewCell.self, forCellReuseIdentifier: "com.testApp.reuseIdentifier")
+        tableView.separatorStyle = .None
+    }
+
+    override func viewDidAppear(animated: Bool) {
+        super.viewDidAppear(animated)
+        initialNavigationControllerDelegate = navigationController?.delegate
+    }
+
+    // MARK: UITableViewDataSource
+
+    override func numberOfSectionsInTableView(tableView: UITableView) -> Int {
+        return 1
+    }
+    override func tableView(tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return 10
+    }
+
+    override func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCellWithIdentifier("com.testApp.reuseIdentifier", forIndexPath: indexPath)
+        cell.selectionStyle = .None
+        cell.backgroundColor = colors[indexPath.row]
+        return cell
+    }
+
+    override func tableView(tableView: UITableView, heightForRowAtIndexPath indexPath: NSIndexPath) -> CGFloat {
+        return 100.0
```

Comment:
It's more performant to set `tableView.rowHeight` when all the cells have the same height.